### ISSUE TITLE
Gunicorn keep booting as soon as I add one more spacy model

### ISSUE THREAD

**Issue by jinnycho503** [benoitc/gunicorn#1747]
**TL;DR** Hi! As soon as I add French Spacy model (`fr_core_news_md`), Docker stops giving me a response and gunicorn keeps booting workers almost indefinitely and I couldn't find any meaningful way to debug. I'm not sure if it's gunicorn's problem or Docker's problem or something else.

### My Dockerfile
```
FROM python:3.6

# Install the required packages
RUN apt-get update \
 && apt-get install -y \
    build-essential \
    libssl-dev \
    jq \
    curl \
  && apt-get -q clean -y && rm -rf /var/lib/apt/lists/* \
  && rm -f /var/cache/apt/*.bin

RUN pip3 install awscli==1.11.141 \
 && aws configure set default.s3.signature_version s3v4

RUN curl -sL https://github.com/mapbox/decrypt-kms-env/archive/v1.0.6.tar.gz \
    | tar --gunzip --extract --strip-components=1 --exclude=readme.md --directory=/usr/local

WORKDIR /usr/src/app/
COPY requirements.txt /usr/src/app/
RUN pip3 install --no-cache-dir -r ./requirements.txt
RUN pip3 install boto3
RUN pip3 install botocore
RUN pip3 install -U spacy
RUN python -m spacy download en
RUN python -m spacy download en_core_web_lg
RUN python -m spacy download xx_ent_wiki_sm
RUN python -m spacy download es_core_news_md
RUN python -m spacy download fr_core_news_md

COPY ./ /usr/src/app/
ENV PORT=80
EXPOSE 80
CMD ./server.sh
```

### Gunicorn
```
function run_server() {
  echo "Starting API server"
  gunicorn \
      --access-logfile - \
      --access-logformat '"%(r)s" %(s)s %(L)ss' \
      --worker-class gevent \
      --workers 1 \
      --threads 3 \
      --graceful-timeout 80 \
      --timeout 90 \
      --bind "0.0.0.0:$PORT" profanity.api:app
}
```
1. I first build Docker image using the docker file above
2. I run Docker
3. Then I use curl to see if things work. like `curl 'https://...'` 

Everything works with the above parameters.(I get the response) 
But as soon as I add `RUN python -m spacy download fr_core_news_md` in Dockerfile, gunicorn keeps booting workers almost indefnitely. 
<img width="695" alt="screen shot 2018-04-03 at 3 48 42 pm" src="https://user-images.githubusercontent.com/35614147/38521604-2b2ef314-3c14-11e8-998b-7646d000a19b.png">

I wonder if it has something to do with timeout. But increasing timeout to a very big number (for me) didn't really help. I am also not sure if it's a problem of Docker.. Do you have any idea to debug/fix this issue?

**Comment by tilgovi** [comment]
There is very little time between those messages, so it's unlikely to be a timeout. The other thing I would check is memory usage. If the kernel is killing the worker for memory reasons it will keep booting like this.

**Comment by benoitc** [comment]
closing the ticket since no activity since
